pkg/kubelet/kubelet.go

```diff
@@ -1521,6 +1545,12 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 		return fmt.Errorf("no node instance returned for %q", kl.hostname)
 	}
 
+	if kl.configureCBR0 {
+		if err := kl.reconcileCBR0(node.Spec.PodCIDR); err != nil {
+			glog.Errorf("Error configuring cbr0: %v", err)
```

Comment:
When configuring cbr0 failed, shouldn't we change NodeCondition to some bad state, at least NotReady state? So that master won't schedule work to the node? 

cc/ @bgrant0607 to add new NodeCondition for network configure issue.